identity/id.go

```diff
@@ -1,6 +1,7 @@
 package identity
 
 import (
+	"crypto/rand"
```

Comment:
```suggestion
	"math/rand"
```
As `RandomID` is only intended for unit tests, there is no need to use a cryptographic RNG, math/rand is faster and should be sufficient.